### PR TITLE
chore: remove AppCenter related code [WPB-8600]

### PR DESCRIPTION
**Pull request by caldrian**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-8600" title="WPB-8600" target="_blank">StoryWPB-8600</a>  Remove AppCenter
  </td></table>
  <br />
 

### Issue

AppCenter is about to retire, so all code will be removed.

see also: https://github.com/wireapp/wire-ios-build-assets/pull/107

---

### Checklist

- [x] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [x] Description is filled and free of optional paragraphs.
- [ ] Adds/updates automated tests.

### CODE REVIEW

File: wire-ios/Wire-iOS/Sources/Components/Settings.swift [hunk 1]
```diff
@@ -65,7 +65,7 @@ enum SettingKey: String, CaseIterable {
     case twitterOpeningRawValue = "TwitterOpeningRawValue"
     case mapsOpeningRawValue = "MapsOpeningRawValue"
     case browserOpeningRawValue = "BrowserOpeningRawValue"
-    case didMigrateHockeySettingInitially = "DidMigrateHockeySettingInitially"
+    case didMigrateHockeySettingInitially = "DidMigrateHockeySettingInitially" // still needed?
```
Comment (netbe) [inline wire-ios/Wire-iOS/Sources/Components/Settings.swift]: I guess no

File: wire-ios/Wire-iOS/Sources/Components/Settings.swift [hunk 2]
```diff
@@ -152,7 +152,7 @@ class Settings {
         NotificationCenter.default.addObserver(self, selector: #selector(applicationDidEnterBackground(_:)), name: UIApplication.didEnterBackgroundNotification, object: nil)
     }
 
-    func migrateAppCenterAndOptOutSettingsToSharedDefaults() {
+    func migrateAppCenterAndOptOutSettingsToSharedDefaults() { // still needed?
         if !defaults.bool(forKey: SettingKey.didMigrateHockeySettingInitially.rawValue) {
             ExtensionSettings.shared.disableLinkPreviews = Settings.disableLinkPreviews
```
Comment (netbe) [inline wire-ios/Wire-iOS/Sources/Components/Settings.swift]: do we set the disableLinkPreviews else where?

**Review by netbe — COMMENTED** [review]
you'll need also to clean the wire-ios-build-assets

File: wire-ios/Wire-iOS/Sources/Components/Settings.swift (hunk 2 above)
Comment (caldrian) [reply wire-ios/Wire-iOS/Sources/Components/Settings.swift]: `Settings.disableLinkPreviews` reads and writes the value.

**Review by typfel — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/Components/Settings.swift [hunk 3]
```diff
@@ -65,7 +65,6 @@ enum SettingKey: String, CaseIterable {
     case twitterOpeningRawValue = "TwitterOpeningRawValue"
     case mapsOpeningRawValue = "MapsOpeningRawValue"
     case browserOpeningRawValue = "BrowserOpeningRawValue"
-    case didMigrateHockeySettingInitially = "DidMigrateHockeySettingInitially"
```
Comment (typfel) [inline wire-ios/Wire-iOS/Sources/Components/Settings.swift:68]: HockeyApp! That was a long time ago :)

**Review by netbe — APPROVED** [review]

**Pull request merged**